flytekit/core/tracker.py

```diff
@@ -77,7 +122,7 @@ def find_lhs(self) -> str:
             raise _system_exceptions.FlyteSystemException(f"Object {self} does not have an _instantiated in")
 
         logger.debug(f"Looking for LHS for {self} from {self._instantiated_in}")
-        m = _importlib.import_module(self._instantiated_in)
+        m = importlib.import_module(self._instantiated_in)
         for k in dir(m):
```

Comment:
previously if this loop didn't raise anything we would just raise an exception on line 96.  Can you elaborate on the circumstances in which we want to keep going?  is it just to support the case when the task is in __main__?

Reply:
correct, this happens because eager workflows are just using a FlyteRemote object to execute tasks that are in the namespace at execution time. All the logic below is to try to load the module from the `_module_file`, hence we cannot find an exact match because the task functions are no longer equal to the equivalent functions in `main`